p2p/gossip.go

```diff
@@ -21,6 +20,9 @@ type GossipMessage struct {
 // GossipHandler is a callback function type.
 type GossipHandler func(*GossipMessage)
 
+// GossipValidator is a callback function type.
+type GossipValidator func(*GossipMessage) bool
+
```

Comment:
I think some more informative documentation here would go a long way in helping readers distinguish between a `GossipValidator` and a `GossipHandler`.

Reply:
Actually, handler will be removed soon after MVP (see #126).